plugin/pkg/admission/resourcequota/admission.go

```diff
@@ -77,6 +77,9 @@ var resourceToResourceName = map[unversioned.GroupResource]api.ResourceName{
 	api.Resource("resourcequotas"):         api.ResourceQuotas,
 	api.Resource("secrets"):                api.ResourceSecrets,
 	api.Resource("persistentvolumeclaims"): api.ResourcePersistentVolumeClaims,
+	extensions.Resource("replicasets"):     extensions.ResourceReplicaSets,
```

Comment:
does this mean we would be putting unqualified names in a quota object to refer to group-qualified resources? don't we want unambiguous names here?